docker_test.go

```diff
@@ -417,6 +535,16 @@ func run(m *testing.M) (int, error) {
 		return 0, fmt.Errorf("Could not connect to postgres %q: %w", DB_DSN, err)
 	}
 	fmt.Println("DB_DSN:", DB_DSN)
+
+	initWHConfig()
+
+	defer SetWHPostgresDestination(pool)()
```

Comment:
I don't like the fact that we are using and setting globals, what if ` SetWHPostgresDestinatio` was to be `SetupWHPostgresDestinatio` and return a struct (also this struct could have a method for cleanup).

Reply:
@achettyiitr